spec/controllers/transform_results_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TransformResultsController do
+  let(:exhibit) { create(:exhibit) }
+  let(:curator) { create(:exhibit_curator, exhibit: exhibit) }
+
+  before do
+    sign_in curator
+  end
+
+  describe 'GET show' do
+    it 'is successful' do
+      get :show
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'POST create' do
+    let(:mock_transform_result) { instance_double(TransformResult) }
+
+    let(:msg) do
+      '{"Message": "{\n  \"success\": true,\n  \"records\": 1,\n  \"data_path\": \"stanford/maps/data/' \
+      'kj751hs0595.mods\",\n  \"timestamp\": \"2019-02-22T19:04:24+00:00\",\n  \"duration\": 0,\n  \"url\": ' \
+      '\"http://localstack:4572/dlme-transform/output-20190222190423.ndjson\"\n}\n", "Type": "Notification", ' \
+      '"TopicArn": "arn:aws:sns:us-east-1:123456789012:dlme-transform", "MessageId": "8456e5c9-2fcf-4866-9c4f-b5bf3b898938"}'
+    end
+
+    before do
+      allow(TransformResult).to receive(:new).and_return(mock_transform_result)
+      allow(mock_transform_result).to receive(:save)
+    end
+
+    # rubocop:disable RSpec/ExampleLength
+    it 'returns 200' do
+      post :create, body: msg
+      expect(TransformResult).to have_received(:new).with(
```

Comment:
You are checking for `new` and `save` here, but in the actual controller, you're calling `create`.  Any reason?